doc/ipythonconsole.rst

```diff
@@ -43,16 +43,16 @@ To connect to an external kernel,
    If using Spyder 3.3.0 or later, you'll need to do so with ``python -m spyder_kernels.console`` (after you've first installed ``spyder-kernels`` on the host with ``<conda/pip> install spyder-kernels``).
    If using a version of Spyder before 3.3.0, ``ipython kernel`` should work to launch the kernel, albeit without certain Spyder-specific features.
 
-#. Copy the connection file (:file:`{runtime-dir}/kernel-{pid}.json`) to the machine you're running Spyder on (if remote) or note its location (if local).
+#. Copy the connection file (:file:`{jupter/runtime/dir}/kernel-{pid}.json`) to the machine you're running Spyder on (if remote) or note its location (if local).
```

Comment:
jupter -> jupyter

Reply:
The same error is present in your other changes.

Reply:
Well, that's awfully embarrassing. So much for

> [x] Checked your writing carefully for correct English spelling, grammar, etc

Thanks for the catch; fixed.